### ISSUE TITLE
delete confirm modal doesn't work on nova 2.8

### ISSUE THREAD

**Issue by pslxx** [InfinetyEs/Nova-Filemanager#109]

**Comment by pslxx** [comment]
```
<template>
<!--    <portal to="portal-filemanager" name="Remove File" transition="fade-transition" >-->
        <modal @modal-close="handleClose">
            <div
                class="bg-white rounded-lg shadow-lg overflow-hidden"
                style="width: 460px"
            >
                <div class="p-8">
                    <heading :level="2" class="mb-6">{{__('Deselect File')}}</heading>
                    <p class="text-80">{{__('Are you sure you want to deselect this file?')}}</p>
                    <p class="text-sm text-80 mt-2">{{ __('Remember: The file will not be delete from your storage') }}</p>
                </div>

                <div class="bg-30 px-6 py-3 flex">
                    <div class="ml-auto">
                        <button dusk="cancel-upload-delete-button" type="button" data-testid="cancel-button" @click.prevent="handleClose" class="btn text-80 font-normal h-9 px-3 mr-3 btn-link">{{__('Cancel')}}</button>
                        <button dusk="confirm-upload-delete-button" ref="confirmButton" data-testid="confirm-button" @click.prevent="handleConfirm" class="btn btn-default btn-danger">{{__('Deselect')}}</button>
                    </div>
                </div>
            </div>
        </modal>
<!--    </portal>-->
</template>
```

be work

**Comment by Krato** [comment]
@pslxx Thanks! Can you make a PR?

**Comment by Krato** [comment]
Fixed on [v2.6.5](https://github.com/InfinetyEs/Nova-Filemanager/releases/tag/2.6.5)
